doc/howto/autoinstall-quickstart.rst

```diff
@@ -0,0 +1,183 @@
+        -cdrom ~/Downloads/ubuntu-22.10-live-server-amd64.iso \
+        -kernel /mnt/casper/vmlinuz \
+        -initrd /mnt/casper/initrd \
+        -append 'autoinstall ds=nocloud-net;s=http://_gateway:3003/'
+
+This will boot, download the config from the server (set up in the previous
+step) and run the install. The installer reboots at the end but the
+``-no-reboot`` flag to ``kvm`` means that ``kvm`` will exit when this happens.
+It should take about 5 minutes.
+
+Boot the installed system
+-------------------------
+
+.. code-block:: bash
+
+    kvm -no-reboot -m 2048 \
+        -drive file=image.img,format=raw,cache=none,if=virtio
+
+This will boot into the freshly installed system and you should be able to log
+in as ``ubuntu/ubuntu``.
+
+Using another volume to provide the autoinstall config
+======================================================
+
+This is the method to use when you want to create media that you can just plug
+into a system to have it be installed.
+
+Download the live-server ISO
+----------------------------
+
+Go to the `22.10 ISO download page`_ and download the latest Ubuntu 22.10
+live-server ISO.
+
+Create your user-data and meta-data files
+-----------------------------------------
+
+.. code-block:: bash
+
+    mkdir -p ~/cidata
+    cd ~/cidata
+    cat > user-data << 'EOF'
+    #cloud-config
+    autoinstall:
+    version: 1
+    identity:
+        hostname: ubuntu-server
+        password: "$6$exDY1mhS4KUYCE/2$zmn9ToZwTKLhCw.b4/b.ZRTIZM30JZ4QrOQ2aOXJ8yk96xpcCof0kxKwuX1kqLG/ygbJ1f8wxED22bTL4F46P0"
+        username: ubuntu
+    EOF
+    touch meta-data
+
+The crypted password is just ``ubuntu``.
+
+Create an ISO to use as a cloud-init data source
+------------------------------------------------
+
+.. code-block:: bash
+
+    sudo apt install cloud-image-utils
+    cloud-localds ~/seed.iso user-data meta-data
+
+Create a target disk
+--------------------
+
+.. code-block:: bash
+
+    truncate -s 10G image.img
+
+Run the install!
+----------------
+
+.. code-block:: bash
+
+    kvm -no-reboot -m 2048 \
+        -drive file=image.img,format=raw,cache=none,if=virtio \
+        -drive file=~/seed.iso,format=raw,cache=none,if=virtio \
+        -cdrom ~/Downloads/ubuntu-22.10-live-server-amd64.iso
+
+This will boot and run the install. Unless you interrupt boot to add
+'autoinstall' to the kernel command line, the installer will prompt for
+confirmation before touching the disk.
+
+The installer reboots at the end but the ``-no-reboot`` flag to ``kvm`` means
+that ``kvm`` will exit when this happens.
+
+The whole process should take about 5 minutes.
+
+Boot the installed system
+-------------------------
+
+.. code-block:: bash
+
+    kvm -no-reboot -m 2048 \
+        -drive file=image.img,format=raw,cache=none,if=virtio
+
+This will boot into the freshly installed system and you should be able to log
+in as ``ubuntu/ubuntu``.
+
+.. LINKS
+
+.. _22.10 ISO download page: https://releases.ubuntu.com/22.10/
```

Comment:
I'm a big believer in making links version-agnostic if possible. Would it harm the guide if we just used releases.ubuntu.com? Or do we need to use 22.10 specifically?

Reply:
yes, this is a great place for a generic link for releases.ubuntu.com

Reply:
I have taken the liberty of making the entire page release-agnostic. Please take a look and let me know if I've changed anything I shouldn't (or missed anything I should have!)